docs/docs/concepts/infrastructure-architecture.mdx

```diff
@@ -0,0 +1,142 @@
+  can interact with their deployed application.
+
+## Looking Under the Hood
+
+When a user selects a framework, JHub Apps balances the given requests into two possible
+proxies, each one corresponding to a different deployment type:
+- For standard JupyterLab instances, the request is only managed by the default
+[jupyter-server-proxy](https://github.com/jupyterhub/jupyter-server-proxy), which
+launches the JupyterLab instance, under your usual `/lab` URL.
+- For other frameworks, an additional proxy layer is launched within the web-service, the
+[jhsingle-native-proxy](https://github.com/ideonate/jhsingle-native-proxy), which
+works in a similar fashion as the above, but is designed to remove the dependency on the
+jupyter notebook and jupyterlab services, while also allowing for a direct connection to the
+selected framework (webs service) during its lifecycle.
+
+Here’s a simplified explanation of the interaction between the two proxies:
+
+1. **Public Proxy (configurable-http-proxy):**
+   - Acts as a gatekeeper, for all receiving requests.
+   - This proxy decides where each request should go based on the user's information and
+     routes it accordingly.
+2. **Internal Proxies:**
+    - [jupyter-server-proxy](https://github.com/jupyterhub/jupyter-server-proxy):
+      - Jupyter Server Proxy lets you run arbitrary external processes (such as RStudio,
+        Shiny Server, Syncthing, PostgreSQL, Code Server, etc) alongside your notebook
+        server and provide authenticated web access to them using a path like /rstudio
+        next to others like `/lab`.
+    - [jhub-single-native](https://github.com/ideonate/jhsingle-native-proxy):
+      - Similar to the public proxy, it also dynamically updates the available route
+        table, it is, allows a public facing endpoint to be created for the selected
+        web-service (framework), though its main difference is that it is designed to be
+        a lightweight proxy, removing the dependency on the jupyter notebook and
+        jupyterlab services.
+
+### How does all Work Together?
+
+As an example, the diagram below illustrates the interaction between the two given
+proxies (http--jupyter-server-proxy or http--jhub-single-native), in a kubernetes
+environment.
+
+To grasp the general concept of how the routing is handled, you can disregard the
+specifics of the kubernetes environment, such as the `Ingress (Route)`, and `Service
+(Proxy-public)` in the diagram bellow:
+
+![JHub Apps Proxy Interaction](/img/jhub_single_native_proxy.png)
+
+- **Request Handling:**
+  1. A user sends a request to access their selected framework through the JHub Apps
+     home page.
+  2. The public proxy receives the request and forwards it to the Jhub Apps service
+     internally, which then launches the selected web-service through the internal proxy
+     in a containerized environment.
+      - Based on the selected framework, we will end up spawning different proxies, as
```

Comment:
This is the first (and perhaps only) usage of the word "we". Can you swap that out so that the language is consistent throughout?

Reply:
This is a nitpick and if we just want to go ahead and push this through as-is, its fine with me. @aktech

Reply:
I agree, I'll commit your suggestions and get this in.